binding.gyp

```diff
@@ -2,13 +2,22 @@
   'targets': [
     {
       'target_name': 'bufferutil',
-      'include_dirs': ["<!(node -e \"require('nan')\")"],
-      'sources': [ 'src/bufferutil.cc' ],
-      'xcode_settings': {
-        'MACOSX_DEPLOYMENT_TARGET': '10.8',
+      'cflags!': [ '-fno-exceptions' ],
```

Comment:
I think we don't need all this stuff for exception handling.
We only need to add/change:

```
'include_dirs': ["<!@(node -p \"require('node-addon-api').include\")"],
'dependencies': ["<!(node -p \"require('node-addon-api').gyp\")"],
'defines': [ 'NAPI_DISABLE_CPP_EXCEPTIONS' ],
```

Reply:
Okay, I'll disable CPP Exceptions.

Reply:
Is this line and the following still required?